src/frontend/Komponenter/Verktøy/BeregningsskjemaSide/utils.ts

```diff
@@ -135,3 +135,57 @@ const regnUtGjennomsnitt = <Beregning>(
     const sum = beregninger.reduce((acc, property) => acc + selector(property), 0);
     return sum / beregninger.length;
 };
+
+/**
+ * Beregner ny månedlig beregning basert på årslønn, periode og grunnbeløp.
+ *
+ * Excel-utregning:
+ * =IF(NOT(I20="");IF(NOT(R20="");
+ * MAX(ROUND(IF(AND(H20<2017;$BP$2=2);
+ * E20*$BO$3;IF(OR(H20>2016;$BP$2=3);
+ * E20*$BO$4;0))/12;0)-IF(NOT(R20="");
+ * IF(R20>E20/2;ROUND(((R20-(E20/2))/12)*(IF(AND(H20<2017;$BP$2=2);
+ * $BN$3;IF(OR(H20>2016;$BP$2=3);$BN$4;0))/100);0);0);0);0);"");"")
+ *
+ * @param beregning - Beregning-objekt med periode og årslønn
+ * @returns Ny månedlig beregning (number)
+ */
+export const regnUtNyBeregning = (beregning: Beregning): number => {
+    const { måned, årstall: år } = beregning.periode;
+    const årstall = parseInt(år);
+    const årslønn = beregning.årslønn;
+
+    const GRUNNBELØP: number = 130160 /* TODO: bruke api - Grunnbeløp for 2025 */,
+        TO = 2,
+        TO_OG_EN_FJERDEDEL = 2.25,
+        GAMMEL_ORDNING = 40,
+        NY_ORDNING = 45; // ny ordn. (etter 010414)
+
+    if (måned === '') return 0;
+    if (årslønn === 0) return 0;
+
+    let basis = 0;
+    if (årstall < 2017) {
+        basis = GRUNNBELØP * TO;
+    } else if (årstall > 2016) {
+        basis = GRUNNBELØP * TO_OG_EN_FJERDEDEL;
+    }
+
+    const baseMåntlig = Math.round(basis / 12);
```

Comment:
```suggestion
    const baseMånedlig = Math.round(basis / 12);
```
:P

Reply:
Er dette hvor mye bruker skal få utbetalt per måned fremover? Kunne man renamet den til å forklare dette? 
`månedligUtbetaling` for eksempel?